signer-js/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "signer-js",
+  "version": "1.0.0",
+  "description": "",
+  "main": "src/index.js",
+  "scripts": {
+    "test": "mocha"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "base32-decode": "^1.0.0",
+    "base32-encode": "^1.1.1",
+    "bip32": "^2.0.5",
+    "bip39": "^3.0.2",
+    "blake2": "^4.0.1",
+    "borc": "^2.1.2",
+    "secp256k1": "^4.0.1"
+  },
+  "devDependencies": {
+    "mocha": "^7.2.0"
```

Comment:
can we add at least eslint like in the other projects

Reply:
I can add it but do we have a place where we call it ? I believe in the other projects it is integrated with jest or babel but we don't use them here.

Do we want automatic linting or just warning ?